groups/frontend/locals.tf

```diff
@@ -0,0 +1,23 @@
+locals {
+  application_subnet_ids_by_az = values(zipmap(data.aws_subnet.application.*.availability_zone, data.aws_subnet.application.*.id))
+
+  common_tags = {
+    Environment    = var.environment
+    Service        = var.service
+    ServiceSubType = var.service_subtype
+    Team           = "Platform"
```

Comment:
While this is unlikely to change is it worth defining this through a team variable defaulted to `Platform`? It might dissuade copy and paste coders from hardcoding their infrastructure